modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase

```diff
@@ -57,12 +57,82 @@ definition {
 	}
 
 	@description = "This is a test for COMMERCE-6089. RemoveAccountGroupFilter"
-	@ignore = "true"
 	@priority = "3"
 	test RemoveAccountGroupFilter {
 		property portal.acceptance = "false";
 
-		// TODO COMMERCE-6089 RemoveAccountGroupFilter
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Catalogs");
+
+		Click(locator1 = "Button#PLUS");
+
+		CommerceEntry.addNewCatalogsEntry(
+			catalogCurrency = "US Dollar",
+			catalogLanguage = "English (United States)",
+			catalogName = "Master");
+
+		AssertClickNoError(
+			locator1 = "Button#SUBMIT",
+			value1 = "Submit");
+
+		Refresh();
+
+		ApplicationsMenu.gotoPortlet(
+			category = "Users",
+			panel = "Control Panel",
+			portlet = "Account Groups");
+
+		Click(locator1 = "Button#PLUS");
+
+		CommerceEntry.addNewAccountGroupsEntry(entryName = "Guest");
+
+		Button.clickSave();
+
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Products");
+
+		CPCommerceCatalog.newProductsEntry(
+			productCatalog = "Master",
+			productName = "Simple T-Shirt",
+			productType = "Simple");
+
+		CommerceEntry.gotoMenuTab(menuTab = "Visibility");
+
+		Click(
+			key_toggleSwitch = "wiwk___toggle-switch",
+			locator1 = "ToggleSwitch#SPECIFIC_TOGGLE_SWITCH");
+
+		ClickNoError.clickNoMouseOver(
+			key_webContentTitle = "Add Account Group Relation to Simple T-Shirt",
+			locator1 = "Button#BUTTON_PLUS_PRODUCT_VISIBILITY");
```

Comment:
i think is possible to use: `Click(locator1 = "Button#PLUS")`

Reply:
isn't possible.